docs/resources/scm_certificate.md

```diff
@@ -0,0 +1,146 @@
+---
+subcategory: "SSL Certificate Manager (SCM)"
+---
+
+# huaweicloud\_scm\_certificate
+
+SSL Certificate Manager (SCM) allows you to purchase Secure Sockets Layer (SSL) certificates from 
+the world's leading digital certificate authorities (CAs), upload existing SSL certificates, and 
+centrally manage all your SSL certificates in one place.
+
+## Example Usage
+
+```hcl
+# Load the certificate contents from the local files.
+resource "huaweicloud_scm_certificate" "certificate_2" {
```

Comment:
Nits: better to use certificate_1 here and certificate_2 below :D

Reply:
fixed